### DISCUSSION TITLE
argo-workflows/lifecyclehook/

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#10094] [General]
# argo-workflows/lifecyclehook/

https://argoproj.github.io/argo-workflows/lifecyclehook/

**Comment by Rosyya** [comment]
does hook support arguments.parameters?

**Comment by dazinator** [comment]
In this "notification use case"  is the "running" lifecycle hook, functionally the same as adding a step to the start of the workflow that sends the notification instead?

**Comment by shenshanyoumu** [comment]
I want to use steps.<stepname>.exitCode but not works

**Comment by ritiksethi21** [comment]
can anyone help me with how does the workflow.status works internally to fetch the status of workflow?
